python/whylogs/extras/image_metric.py

```diff
@@ -101,32 +102,65 @@ def image_based_metadata(img):
     }
 
 
+@dataclass(frozen=True)
+class ImageMetricConfig(MetricConfig):
+    allowed_exif_tags: Set[str] = field(default_factory=set)
+    forbidden_exif_tags: Set[str] = field(default_factory=set)
```

Comment:
This capability might be interesting, but I don't know of anyone requesting this feature and it complicates the calling pattern. I thought you only had explored using ImageMetricConfig as a work-around to having no support for logging exif tags like we did in v0, and not having ability to copy metrics in a merge. Why do we need ImageMetricConfig?